vector/src/main/res/menu/menu_new_home.xml

```diff
@@ -8,6 +8,11 @@
         android:title="@string/invite_friends"
         app:showAsAction="never" />
 
+    <item
```

Comment:
do we need some logic to determine if the menu item should be visible?

Reply:
Yes! Thanks for this catch, I forgot about it